module-1/cdk/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "cdk",
+  "version": "0.1.0",
+  "bin": {
+    "cdk": "bin/cdk.js"
+  },
+  "scripts": {
+    "lint": "cdk-lint",
+    "test": "../../tools/cdk-build-tools/bin/cdk-test",
+    "integ": "cdk-integ",
+    "awslint": "cdk-awslint",
+    "cfn2ts": "cfn2ts",
+    "build": "tsc",
+    "watch": "tsc -w",
+    "cdk": "cdk"
+  },
+  "devDependencies": {
+    "@aws-cdk/assert": "^0.33.0",
+    "@types/node": "12.0.4",
+    "@types/nodeunit": "0.0.30",
```

Comment:
We prefer jest instead of the deprecated nodeunit nowadays.

Reply:
https://github.com/DavidChristiansen/aws-modern-application-workshop/issues/32